### PR TITLE
Basic xkit-react setup

### PR DESCRIPTION
**Pull request by kozlovzxc-at-xkit**
:heavy_check_mark: Basic app
:heavy_check_mark: Readme

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Xkit React</title>
+</head>
+<body>
+    <div id="root"></div>
+    <script src="./index.tsx"></script>
+</body>
+</html>
```
Comment (treygriffith) [inline src/index.html]: missing newline

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+    "compilerOptions": {
+        "allowSyntheticDefaultImports": true
+    },
+    "include": [
+        "src/**/*.{ts,tsx,d.ts}"
+    ]
+}
```
Comment (treygriffith) [inline tsconfig.json]: missing newline

File: src/index.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+import './index.css';
+import { App } from './app';
+
+ReactDOM.render(
+  <StrictMode>
+    <App />
+  </StrictMode>,
+  document.getElementById('root')
+);
+
```
Comment (treygriffith) [inline src/index.tsx]: extra newline?

**Review by treygriffith — APPROVED** [review]

**Comment by kozlovzxc-at-xkit** [comment]
@treygriffith added prettier -> fixed these newlines

**Pull request merged**